### PR TITLE
Scc 4168/chris qa1

### PR DESCRIPTION
**Pull request by charmingduchess**
## Ticket:

- JIRA ticket [SCC-4168](https://newyorkpubliclibrary.atlassian.net/browse/SCC-4168)

## This PR does the following:

- sort checkouts and holds into chronological order
- append authors to titles
- updates hold and checkout fixtures, as well as refactoring some tests to be more flexible to differently ordered fixtures
- removes a bunch of unused imports and vars (sorry for the noise)

## Accessibility concerns or updates

-

### Checklist:

- [ ] I updated the CHANGELOG with the appropriate information and JIRA ticket number (if applicable).
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/MyAccount/CheckoutsTab/CheckoutsTab.test.tsx [hunk 1]
```diff
@@ -33,11 +33,14 @@ describe("CheckoutsTab", () => {
     const component = render(
       <CheckoutsTab patron={processedPatron} checkouts={processedCheckouts} />
     )
+    const renewableCheckout = processedCheckouts[0]
+    const row = component.getByText(renewableCheckout.title).closest("tr")
```
Comment (EdwinGuzman) [inline src/components/MyAccount/CheckoutsTab/CheckoutsTab.test.tsx:37]: I just noticed this does `component.getBy...` where `component` is coming from the render function. Usually, it's  `screen.getBy...` and screen is imported from the package. Did the latter not work and you had to use `component` from the render function? They're essentially the same so more out of curiosity.

File: src/models/MyAccount.ts [hunk 2]
```diff
@@ -154,12 +154,25 @@ export default class MyAccount {
     return bibs.reduce((bibDataMap: BibDataMapType, bibFields) => {
       const { isResearch, isNyplOwned } =
         this.getResearchAndOwnership(bibFields)
-      const title = bibFields.title
+      const title = `${bibFields.title}${
+        bibFields.author && isNyplOwned ? ` / ${bibFields.author}` : ""
```
Comment (EdwinGuzman) [inline src/models/MyAccount.ts:158]: I'm guessing that the date + "- author" is a data problem?

Screenshot 2024-06-26 at 2 48 45 PM

**Review by EdwinGuzman — APPROVED** [review]
Just one question

File: src/components/MyAccount/CheckoutsTab/CheckoutsTab.test.tsx (hunk 1 above)
Comment (charmingduchess) [reply src/components/MyAccount/CheckoutsTab/CheckoutsTab.test.tsx:37]: No this was inadvertent. I'll update.

Comment (charmingduchess) [reply src/components/MyAccount/CheckoutsTab/CheckoutsTab.test.tsx:37]: Ok, actually... Component is returned by render which comes from our wrapped methods. I'm not exactly sure why that's the pattern here, but it's working, and I'm not so inclined to fix it. Unless you think its important to have it standardized.

File: src/models/MyAccount.ts (hunk 2 above)
Comment (charmingduchess) [reply src/models/MyAccount.ts:158]: Yes. I think some author fields have the years the author is alive for, and also the word author apparently.

File: src/components/MyAccount/CheckoutsTab/CheckoutsTab.test.tsx (hunk 1 above)
Comment (EdwinGuzman) [reply src/components/MyAccount/CheckoutsTab/CheckoutsTab.test.tsx:37]: ah, right. it's ok to keep

**Pull request merged**